tests/dummy/lib/ember-blog/addon/routes/post.js

```diff
@@ -13,6 +13,20 @@ export default Ember.Route.extend({
   actions: {
     goToChineseVersion() {
       this.transitionTo({ queryParams: { lang: 'Chinese' } });
+    },
+    transitionToHome() {
+      this.transitionToExternal('home')
+        .then(() => {
+          var postController = this.controllerFor(this.routeName);
+          postController.set('transitionedToExternal', true);
```

Comment:
I'd prefer we grab the controller for `home` and set a property on it so that we can avoid needing to `visit()` calls in the tests.